test/functional/fixtures/regression/gh-628/test.js

```diff
@@ -0,0 +1,19 @@
+var expect = require('chai').expect;
+
+describe('[Regression](GH-628)', function () {
+    it('Should click on an "option" element', function () {
+        return runTests('./testcafe-fixtures/click-on-select-child-test.js', 'Click on an "option" element');
+    });
+
+    it('Should fail if try to click on an "option" element in a closed "select" element', function () {
+        return runTests('./testcafe-fixtures/click-on-select-child-test.js', 'Click on an invisible "option" element', {
+            shouldFail: true
+        })
```

Comment:
Nah, just keep it as is. it will be screwed up by WebStorm auto-formatting anyway